OmiseSDK/AtomeFormViewController.swift

```diff
@@ -0,0 +1,241 @@
+    
+    @IBOutlet var contentView: UIScrollView!
+    
+    @IBOutlet private var nameTextField: OmiseTextField!
+    @IBOutlet private var emailTextField: OmiseTextField!
+    @IBOutlet private var phoneNumberTextField: OmiseTextField!
+    @IBOutlet private var shippingStreetTextField: OmiseTextField!
+    @IBOutlet private var shippingCityTextField: OmiseTextField!
+    @IBOutlet private var shippingCountryCodeTextField: OmiseTextField!
+    @IBOutlet private var shippingPostalCodeTextField: OmiseTextField!
+    @IBOutlet private var submitButton: MainActionButton!
+    @IBOutlet private var requestingIndicatorView: UIActivityIndicatorView!
+    
+    @IBOutlet private var errorLabel: UILabel!
+    
+    @IBOutlet var formLabels: [UILabel]!
+    @IBOutlet var formFields: [OmiseTextField]!
+    
+    @IBOutlet var formFieldsAccessoryView: UIToolbar!
+    @IBOutlet var gotoPreviousFieldBarButtonItem: UIBarButtonItem!
+    @IBOutlet var gotoNextFieldBarButtonItem: UIBarButtonItem!
+    @IBOutlet var doneEditingBarButtonItem: UIBarButtonItem!
+    
+    // need to refactor loadView, removing super results in crash
+    // swiftlint:disable prohibited_super_call
+    override func loadView() {
+        super.loadView()
+        
+        view.backgroundColor = .background
+        formFieldsAccessoryView.barTintColor = .formAccessoryBarTintColor
+        
+        submitButton.defaultBackgroundColor = .omise
+        submitButton.disabledBackgroundColor = .line
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        applyPrimaryColor()
+        applySecondaryColor()
+        navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
+        
+        formFields.forEach {
+            $0.inputAccessoryView = formFieldsAccessoryView
+        }
+        
+        formFields.forEach {
+            $0.adjustsFontForContentSizeCategory = true
+        }
+        formLabels.forEach {
+            $0.adjustsFontForContentSizeCategory = true
+        }
+        submitButton.titleLabel?.adjustsFontForContentSizeCategory = true
+        
+        if  #available(iOS 11, *) {
+            // We'll leave the adjusting scroll view insets job for iOS 11 and later to the layoutMargins + safeAreaInsets here
+        } else {
+            automaticallyAdjustsScrollViewInsets = true
+        }
+
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(keyboardWillChangeFrame(_:)),
+            name: NotificationKeyboardWillChangeFrameNotification,
+            object: nil
+        )
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(keyboardWillHide(_:)),
+            name: NotificationKeyboardWillHideFrameNotification,
+            object: nil
+        )
+
+        phoneNumberTextField.validator = try? NSRegularExpression(pattern: "\\d{10,11}\\s?", options: [])
```

Comment:
Why do we have to have `\\s?` na? 🤔

Reply:
In case user may have zero or one whitespace at the end, and we also trim phone string before assign value, somehow   we can allow only decimal digit. I will fix this one ka. thank you, bro! :joy: